recipe/meta.yaml

```diff
@@ -25,29 +25,27 @@ build:
 # OSX. We build against the openblas available in conda-forge which includes
 # lapack on Linux and OSX. blas and lapack are not needed on Windows because
 # Simbody includes them. For the visualizer, FreeGLUT is available via
-# anaconda.org and conda-forge but libXi and libXmu are not, so the user will
-# need those to be installed via their Linux package manager. Glut is included
-# with Simbody for Windows and OSX includes a Glut implementation. TODO:
-# Doxygen is the only dependency that is omitted here, so the rendered
-# documentation is not built.
+# anaconda.org. Glut is included with Simbody for Windows and OSX includes a
+# Glut implementation.
+# https://github.com/AnacondaRecipes/r-rgl-feedstock/blob/master/recipe/meta.yaml#L48-L67
 requirements:
     build:
-        - cmake >=2.8.6
+        - cmake >=3.1.3
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+          # - doxygen                 # [not linux]
     host:
         - blas 1.1 {{ variant }}  # [not win]
-        - openblas                        # [not win]
```

Comment:
Is this no longer required because `{{ compiler('c')}}` handles it? If so, can `blas 1.1 {{ variant}}` be removed?

Reply:
Oh sorry, I see it was added back below.